composer.json

```diff
@@ -106,6 +106,9 @@
             },
             "drupal/views_infinite_scroll" : {
                 "Headers in table format repeat on load more instead of adding rows": "https://www.drupal.org/files/issues/2019-08-15/table_tbody_append-2899705-26.patch"
+            },
+            "drupal/graphql": {
+                "Add the ability to get absolute URL of entity": "https://patch-diff.githubusercontent.com/raw/drupal-graphql/graphql/pull/1220.diff"
```

Comment:
Great work on making the contribution to the GraphQL module here! However, while that's still under review, lets create a copy of the Data Producer in Open Social with a different ID and a `@deprecated Remove when #PR is merged`. This allows us to use it in Open Social already, but ensures we don't get patch conflicts when GraphQL 4.2 is removed. Then we can clean up our own code when we upgrade to 4.2.